src/CloudEvents/Serializer/Exception/UnsupportedVersionException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace CloudEvents\Serializer\Exception;
+
+/**
+ * Thrown when the serializer does not support the spec version of the CloudEvent instance.
+ */
+class UnsupportedVersionException extends SerializationException
+{
+
```

Comment:
We should override the `$message` property with a good default message to simplify usage.

```suggestion
    protected string $message = 'Unsupported CloudEvent spec version';
```